string chess_square_pick_pawn
string chess_square_pawn_to_trash
string chess_square_pick_queen
string chess_square_place
---
bool result